### PR TITLE
Adding provider scope example

### PR DESCRIPTION
**Pull request by GiantPineappleStatue**
## What kind of change does this PR introduce?

Doc update. Current Auth UI section does not contain example for providerScope.

## What is the current behavior?

https://supabase.com/docs/guides/auth/auth-helpers/auth-ui

## What is the new behavior?

Added example with correct syntax provided by Supabase support via email.
`<Auth
   supabaseClient={supabase}
   providers={['google']}
   providerScopes={{
     google: 'https://www.googleapis.com/auth/calendar.readonly'
   }}
/>`

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Comment by dshukertjr** [comment]
Hi @GiantPineappleStatue, it seems like prettier check is failing, but could you run prettier on the file you edited?

File: apps/docs/pages/guides/auth/auth-helpers/auth-ui.mdx [hunk 1]
```diff
@@ -96,6 +96,25 @@ Options are available via `queryParams`:
...
+<Auth
+  supabaseClient={supabase}
+  providers={['google']}
+  queryParams={{
+    access_type: 'offline',
+    prompt: 'consent',
+    hd: 'domain.com',
+  }}
+  providerScopes={{
+  google: 'https://www.googleapis.com/auth/calendar.readonly'
```
Comment (awalias) [inline apps/docs/pages/guides/auth/auth-helpers/auth-ui.mdx:113]:
```suggestion
    google: 'https://www.googleapis.com/auth/calendar.readonly'
```

File: apps/docs/pages/guides/auth/auth-helpers/auth-ui.mdx [hunk 2]
```diff
@@ -96,6 +96,25 @@ Options are available via `queryParams`:
...
+  supabaseClient={supabase}
+  providers={['google']}
+  queryParams={{
+    access_type: 'offline',
+    prompt: 'consent',
+    hd: 'domain.com',
+  }}
+  providerScopes={{
+    google: 'https://www.googleapis.com/auth/calendar.readonly'
+}}
```
Comment (awalias) [inline apps/docs/pages/guides/auth/auth-helpers/auth-ui.mdx:114]:
```suggestion
  }}
```

**Review by awalias — APPROVED** [review]

File: apps/docs/pages/guides/auth/auth-helpers/auth-ui.mdx [hunk 3]
```diff
@@ -96,6 +96,25 @@ Options are available via `queryParams`:
...
+<Auth
+  supabaseClient={supabase}
+  providers={['google']}
+  queryParams={{
+    access_type: 'offline',
+    prompt: 'consent',
+    hd: 'domain.com',
+  }}
+  providerScopes={{
+    google: 'https://www.googleapis.com/auth/calendar.readonly'
```
Comment (awalias) [inline apps/docs/pages/guides/auth/auth-helpers/auth-ui.mdx:113]:
```suggestion
    google: 'https://www.googleapis.com/auth/calendar.readonly',
```

**Comment by lhotwll217** [comment]
How do we add multiple scopes? I need Spreadsheet and Drive, ideally the docs would demonstrate both.

**Comment by dshukertjr** [comment]
@lhotwll217 You can pass multiple scopes as a space-separated-string as per the [docs of .signInWithOAuth method](https://supabase.com/docs/reference/javascript/auth-signinwithoauth).

**Pull request merged**